specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -359,24 +435,28 @@ namespace Microsoft.Windows.PushNotifications
     };
 
     [flags]
-    enum PushNotificationRegistrationKind
+    enum PushNotificationRegistrationOptions
```

Comment:
Consider a name like `PushNotificationActivators` or similar. It's no longer general options about registering activation, it's "which kind of activator does a push notification for this registration use."